app/src/main/java/org/eyeseetea/malariacare/data/remote/SdkController.java

```diff
@@ -104,13 +104,13 @@ public static void postFinish() {
 
     public static boolean finishPullJob() {
```

Comment:
@xurxodev  The Job not exists in the new sdk, i think we can remove it now. thanks